great_expectations/render/renderer/content_block/bullet_list_content_block.py

```diff
@@ -107,67 +150,243 @@ def expect_column_unique_value_count_to_be_between(cls, expectation, column_name
                 }
             }]
 
+    # NOTE: This method is a pretty good example of good usage of `params`.
+    # NOTE: I think `column_name` should be deprecated.
     @classmethod
     def expect_column_values_to_be_between(cls, expectation, column_name=""):
-        if (expectation["kwargs"]["min_value"] is None) and (expectation["kwargs"]["max_value"] is None):
+        params = substitute_none_for_missing(
+            expectation["kwargs"],
+            ["column", "min_value", "max_value", "mostly"]
+        )
+
+        if (params["min_value"] is None) and (params["max_value"] is None):
+            # Note: I'm not sure why we're using a different pattern for templating column names...
+            # This would probably be better as `"$column has a bogus $expectation_name expectation.",`
+
+            # Related issue: are we going to want to allow for different styles of parameters?
+            # For example, I could imagine wanting one color of syntax highlighting for variable names,
+            # and a different color for expectations.
+            # In that case, perhaps returning something like this would work?
+            # {
+            #     "template": "$column has a bogus $expectation_name expectation.",
+            #     "params": {
+            #         "column": "my_column",
+            #         "expectation_name": "expect_column_values_to_be_between",
+            #     },
+            #     "param_types": {
+            #         "column": "variable"
+            #         "expectation_name": "expectation"
+            #     }
+            # }
+            # Thoughtfully enumerating values for the param_types k/v object will be important.
             return [{
                 "template": column_name + " has a bogus $expectation_name expectation.",
                 "params": {
                     "expectation_name": "expect_column_values_to_be_between"
                 }
             }]
 
-        if "mostly" in expectation["kwargs"]:
-            if expectation["kwargs"]["min_value"] is not None and expectation["kwargs"]["max_value"] is not None:
+        if "mostly" in params:
+            if params["min_value"] is not None and params["max_value"] is not None:
                 return [{
                     "template": column_name + " must be between $min and $max at least $mostly% of the time.",
```

Comment:
template should change too, right? (min ->min_value, e.g.)

Reply:
yes. Yikes. This is incompletely migrated. Sorry about that.